kanidmd/src/lib/idm/server.rs

```diff
@@ -378,7 +378,22 @@ impl<'a> IdmServerWriteTransaction<'a> {
                 };
 
                 let (auth_session, state) = if is_valid {
-                    AuthSession::new(au, account, &init.appid, self.webauthn, ct)
+                    //TODO: we can keep a cached map of the badlist, and pass by reference rather than by value
```

Comment:
Can you open an issue for this too and put the issue number next to the todo so we don't forget?

Reply:
Opened at #397